src/components/enterprise-user-subsidy/enterprise-offers/tests/EnterpriseOffersBalanceAlert.test.jsx

```diff
@@ -10,21 +10,24 @@ import {
   NO_BALANCE_ALERT_TEXT,
 } from '../data/constants';
 
-const EnterpriseOffersBalanceAlertWrapper = ({
+function EnterpriseOffersBalanceAlertWrapper({
   enterpriseConfig = {
     adminUsers: [],
   },
   hasNoEnterpriseOffersBalance,
-}) => (
-  <AppContext.Provider value={{
-    enterpriseConfig,
-  }}
-  >
-    <EnterpriseOffersBalanceAlert
-      hasNoEnterpriseOffersBalance={hasNoEnterpriseOffersBalance}
-    />
-  </AppContext.Provider>
-);
+}) {
+  return (
+    // eslint-disable-next-line react/jsx-no-constructed-context-values
```

Comment:
Similar question about being consistent on how we're handling this ESLint rule (in some test files, you're using useMemo, but in others like here, you're not).